users/templates/partials/popup_message.html

```diff
@@ -0,0 +1,7 @@
+{% for message in messages %}
```

Comment:
Why two identical partial templates? Why not used common place to place one of them? Or am I blind and they are different? :p

Reply:
Done. Hope now it is proper solution. I moved that to `users/templates/common/`